### DISCUSSION TITLE
"Pipeable Operators" link is broken in the docs

### DISCUSSION THREAD

**Discussion by Manish-Giri** [ReactiveX/rxjs#6492] [Report issues other than bug]
1. Visit the "ES6 via NPM" [guide](https://rxjs.dev/guide/installation#es6-via-npm)
2. Click on "[See Pipeable Operators for more information](https://rxjs.dev/guide/v6/pipeable-operators)"
3. Link is broken

**Comment by raymonddavis** [comment]
The current link points to `https://rxjs.dev/guide/v6/pipeable-operators`.
I think it should go here instead `https://rxjs.dev/guide/operators` right?

**Reply by Manish-Giri** [reply]
Or even more specifically - `https://rxjs.dev/guide/operators#piping`
